crates/language/src/target_language.rs

```diff
@@ -371,6 +374,61 @@ impl PatternLanguage {
     }
 }
 
+impl ValueEnum for PatternLanguage {
+    fn value_variants<'a>() -> &'a [Self] {
+        &[
+            PatternLanguage::JavaScript,
+            PatternLanguage::TypeScript,
+            PatternLanguage::Tsx,
+            PatternLanguage::Html,
+            PatternLanguage::Css,
+            PatternLanguage::Json,
+            PatternLanguage::Java,
+            PatternLanguage::CSharp,
+            PatternLanguage::Python,
+            PatternLanguage::MarkdownBlock,
+            PatternLanguage::MarkdownInline,
+            PatternLanguage::Go,
+            PatternLanguage::Rust,
+            PatternLanguage::Ruby,
+            PatternLanguage::Solidity,
+            PatternLanguage::Hcl,
+            PatternLanguage::Yaml,
+            PatternLanguage::Sql,
+            PatternLanguage::Vue,
+            PatternLanguage::Toml,
+            PatternLanguage::Universal,
+        ]
+    }
```

Comment:
this function already exists see `enumerate` on line 308. feel free to make it return a static ref instead of a vector.
```suggestion
```